src/test/java/uk/gov/companieshouse/web/accounts/transformer/smallfull/directorsreport/impl/SecretaryTransformerImplTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.companieshouse.web.accounts.transformer.smallfull.directorsreport.impl;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.companieshouse.api.model.accounts.directorsreport.SecretaryApi;
+import uk.gov.companieshouse.web.accounts.model.directorsreport.AddOrRemoveDirectors;
+import uk.gov.companieshouse.web.accounts.transformer.smallfull.directorsreport.SecretaryTransformer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+@ExtendWith(MockitoExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class SecretaryTransformerImplTest {
+
+    private SecretaryTransformer secretaryTransformer = new SecretaryTransformerImpl();
```

Comment:
This should be similar to how you have done in the Controller test or service test.
so 

`@InjectMocks private SecretaryTransformerImpl secretaryTransformer;`

Reply:
Done